Dockerfiles/midstream/run_tests.py

```diff
@@ -4,45 +4,71 @@
 import json
 import unittest
 import subprocess
+import logging
 from os import path
 
 
+TEST_WORKING_DIR= os.getcwd()
+# global variables set
+OPERATOR_DIR=TEST_WORKING_DIR+"/operator_dir/"
+OPERATOR_WORK_DIR=TEST_WORKING_DIR+"/test_operator_work_dir/"
+WORK_DIR=TEST_WORKING_DIR+"/output/"
+PLAYBOOKS_DIR="/operator-test-playbooks/"
+ERROR_MESSAGE_PATH=TEST_WORKING_DIR+"/.errormessage"
+logging.basicConfig(filename=ERROR_MESSAGE_PATH)
+
 class RunMidstreamCVPTests():
 
     def setUp(self):
-        self.operator_dir = os.getenv('OPERATOR_DIR',
-                                      "/project/operator_dir/")
-        self.operator_work_dir = os.getenv('OPERATOR_WORK_DIR',
-                                           "/project/test_operator_work_dir/")
-        self.work_dir = os.getenv('WORK_DIR',
-                                  "/project/output/")
-        self.playbooks_dir = os.getenv('PLAYBOOKS_DIR',
-                                       "/project/operator-test-playbooks/")
+        # IMAGE_TO_TESTis an environment variable that takes 
+        # operator-bundle-image 
         self.image_to_test = os.getenv('IMAGE_TO_TEST')
-        self.umoci_bin_path = os.getenv('UMOCI_BIN_PATH',
-                                        '/usr/local/bin/umoci')
+        # VERBOSITY is verbosity of output log range varying from 1 to 4
+        self.verbosity = int(os.getenv('VERBOSITY', '1'))*"v"
+
+    def run_subprocess_command(self, exec_cmd):
+        print(exec_cmd)
+        env = os.environ.copy()
+        env["ANSIBLE_CONFIG"] = "/operator-test-playbook/Dockerfiles/midstream/"
+        env["ANSIBLE_LOCAL_TEMP"] = "/tmp/"
```

Comment:
Why do we set these env variables in Dockerfile if we override them here?

Reply:
These are the environment variables required to set where the ansible temporary files are created. 
By default inside the container it was routed to root directory which we would like to avoid.

Reply:
I think its not necessary. Am removing them.